apps/storybook/package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "@h5web/lib": "workspace:*",
     "@h5web/shared": "workspace:*",
-    "@react-hookz/web": "12.3.0",
+    "@react-hookz/web": "13.1.0",
```

Comment:
Upgrade for the new `useRafHook`, which avoids triggering more renders than the browser can actually process from `onPointerMove`: https://github.com/react-hookz/web/releases/tag/v13.1.0